### PR TITLE
Create anna-new-york.md

### PR DESCRIPTION
**Pull request by annazelikman**

### CODE REVIEW

File: docs/anna-new-york.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# New York
+
+- Big Apple
+- Famous for pizza and Broadway shows
```
Comment (githubteacher) [inline docs/anna-new-york.md:4]: @annazelikman what is the best place to get real New York pizza?

Comment (nightrose79) [inline docs/anna-new-york.md:4]: @annazelikman testing the comment funjction

Comment (annazelikman) [inline docs/anna-new-york.md:4]: just give me a call and I'll buy you a slice

Comment (annazelikman) [inline docs/anna-new-york.md:4]: @zsvajnagi - good to see you in class!

**Comment by githubteacher** [comment]
Looks great @annazelikman :+1:

File: docs/anna-new-york.md (hunk 1 above)
Comment (zsvajnagi) [inline docs/anna-new-york.md:4]: @annazelikman - yeah it's good to see familiar "faces", little late here in London, but manageable.

**Pull request merged**